libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/SfapApiGenerations.kt

```diff
@@ -0,0 +1,79 @@
+package com.salesforce.androidsdk.auth
```

Comment:
I should probably go to the com.salesforce.androidsdk.rest folder. We have a few request/response classes there already.

Reply:
Nice.  Check that out in 👉🏻 https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/2644/commits/2960b1a6f2093267ebaff6be974664fed83d0bde